### DISCUSSION TITLE
How to source control Spoke projects?

### DISCUSSION THREAD

**Discussion by davegoopot** [mozilla/hubs#2953] [General Questions & Discussion] [answered]
What's the recommended setup for a small team, 2-4 people, to collaborate on building a Spoke project? Ideally would store some "source code" in git.

**Comment by robinkwilson** [comment, marked as answer]
Answered here: https://github.com/mozilla/hubs/discussions/2720

Hope this helps!

**Reply by davegoopot** [reply]
That helps for the assets - basically "use Sketchfab".

I'm going to have a go controlling exported "Legacy Spoke" files in git. Starting out that seems to work. I'm sure they will be a gotcha at some point.
